docs/tools/mbed_cli_2/intro.md

```diff
@@ -0,0 +1,7 @@
+# Mbed CLI 2
+
+Starting with version 6.x, Mbed OS is moving to Mbed CLI 2. It uses Ninja as a build system, and CMake to generate the build environment and manage the build process in a compiler-independent manner.
```

Comment:
I believe the x should be 5. The first Mbed OS release with CMake support is Mbed OS 6.5.

```diff
- 6.x
+ 6.5
```